dbt_subprojects/tokens/models/tokens/sonic/tokens_sonic_erc20.sql

```diff
@@ -12,8 +12,8 @@ SELECT
     , symbol
     , decimals
 FROM (VALUES
-      (0x309C92261178fA0CF748A855e90Ae73FDb79EBc7, 'WETH', 18)
-    , (0x039e2fB66102314Ce7b64Ce5Ce3E5183bc94aD38, 'WS', 18)
+      (0x50c42dEAcD8Fc9773493ED674b675bE577f2634b, 'WETH', 18)
```

Comment:
fyi @Hosuke @IrishLatte19 
looks like this is the verified `WETH` contract address on sonic
https://sonicscan.org/token/0x50c42dEAcD8Fc9773493ED674b675bE577f2634b